src/sage/groups/braid.py

```diff
@@ -446,11 +452,18 @@ def permutation(self, W=None):
             sage: B.<s0,s1,s2> = BraidGroup()
             sage: S = SymmetricGroup(4)
             sage: b = s0*s1/s2/s1
-            sage: b.permutation(W=S)
+            sage: c0 = b.permutation(W=S); c0
             (1,4,2)
-            sage: c = b.permutation(W=Permutations(4)); c
+            sage: c1 = b.permutation(W=Permutations(4)); c1
             [4, 1, 3, 2]
-            sage: c == b.permutation()
+            sage: c1 == b.permutation()
+            True
+
+        From a permutation it is also possible to recover the permutation braid::
```

Comment:
```suggestion
        It also applies the canonical section to a permutation::
```
The current version implies a certain uniqueness that is simply not true. Consider $s^3$ versus $s$ for a simple transposition $s$.

Reply:
I put another sentence. I really want to keep the expression `permutation braid` (or `positive permutation braid`) since is a common concept in braid group theory. Please check it.